### PR TITLE
[helm] Add the force_update arg

### PR DESCRIPTION
**Pull request by mor120**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1648
Depends-On: https://github.com/ansible-collections/kubernetes.core/pull/522

##### SUMMARY
Sometimes a Helm repo needs to be updated with a new URL. The `helm repo add` command allows for this with the `--force-update` flag:
```
      --force-update               replace (overwrite) the repo if it already exists
```

##### ISSUE TYPE
- Feature Pull Request - Closes https://github.com/ansible-collections/kubernetes.core/issues/491

##### COMPONENT NAME
`kubernetes.core.helm_repository`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by mor120** [comment]
Going to write test and validate.

File: plugins/modules/helm_repository.py [hunk 1]
```diff
@@ -97,6 +97,13 @@
     type: path
     aliases: [ ssl_ca_cert ]
     version_added: "2.3.0"
+  force_update:
+    description:
+    - Whether or not to replace (overwrite) the repo if it already exists.
+    type: bool
+    aliases: [ force ]
+    default: False
+    version_added: "2.3.3"
```
Comment (mor120) [inline plugins/modules/helm_repository.py:106]: What version should I set here?

File: plugins/modules/helm_repository.py [hunk 2]
```diff
@@ -275,14 +292,15 @@ def main():
         helm_cmd = delete_repository(helm_cmd, repo_name)
         changed = True
     elif repo_state == "present":
-        if repository_status is None:
+        if repository_status is None or force_update:
```
Comment (mor120) [inline plugins/modules/helm_repository.py:295]: My concern is that even if the specs are exactly the same, we will still mark it as changed, on the other side, in helm it works the same, for example:
```
➜ helm repo add stable https://kubernetes.github.io/ingress-nginx
"stable" already exists with the same configuration, skipping
➜ helm repo add stable https://kubernetes.github.io/ingress-nginx --force-update
"stable" has been added to your repositories
```

File: tests/integration/targets/helm/tasks/main.yml [hunk 3]
```diff
@@ -4,4 +4,4 @@
   loop_control:
     loop_var: helm_version
   with_items:
-    - "v3.2.4"
+    - "v3.7.0"
```
Comment (mor120) [inline tests/integration/targets/helm/tasks/main.yml:7]: To get the tests passed.

File: plugins/modules/helm_repository.py (hunk 1 above)
Comment (gravesm) [reply plugins/modules/helm_repository.py:121]: You should use `2.4.0` here.

Comment (mor120) [reply plugins/modules/helm_repository.py]: Done, thanks.

File: plugins/modules/helm_repository.py (hunk 2 above)
Comment (mor120) [reply plugins/modules/helm_repository.py:302]: Alright, thanks.

**Review by gravesm — APPROVED** [review]

**Comment by gravesm** [comment]
regate

**Comment by goneri** [comment]
recheck

**Pull request merged**